recipes/trading-calendars/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This license file is already included in the source distribution, so please remove it here